tools/generate-api-docs.py

```diff
@@ -211,11 +212,21 @@ def mkdir_p(path):
 
 DEBUG = args.debug
 
+if len(args.files) == 0:
+    if DEBUG:
+      args.files.insert(0, "https://raw.githubusercontent.com/opentx/opentx/projectkk2glider/autogen_lua_docs_2/radio/src/lua_api.cpp")
+    else:
+      args.files.insert(0, "https://raw.githubusercontent.com/opentx/opentx/master/radio/src/lua_api.cpp")
+
 for fileName in args.files:
   logDebug("Opening %s" % fileName)
-  with open(fileName, "r") as inp:
-    data = inp.read()
-    parseSource(data)
+  if fileName.startswith("http"):
+    logDebug("Open using urllib2")
+    inp = urllib2.urlopen(fileName)
+  else:
+    inp = open(fileName, "r")
+  data = inp.read()
+  parseSource(data)
```

Comment:
This is a nitpicking, but since you removed the `with` statement, the `inp` needs closing now. So `inp.close()` should be added here.